packages/@aws-cdk/aws-route53/README.md

```diff
@@ -83,7 +83,19 @@ Constructs are available for A, AAAA, CAA, CNAME, MX, NS, SRV and TXT records.
 Use the `CaaAmazonRecord` construct to easily restrict certificate authorities
 allowed to issue certificates for a domain to Amazon only.
 
-### Adding records to existing hosted zones
+### Imports
+
+If you don't know the ID of a Hosted Zone, you can use the `HostedZone.fromLookup`
+to discover and import it:
+
+Note that this requires an explicit environment to query an AWS account. Check
+out the [documentation](https://docs.aws.amazon.com/cdk/latest/guide/environments.html) for an overview of environments and how to implement.
```

Comment:
perhaps we should include an example ourselves showing basic usage

```suggestion
⚠️ `fromLookup` requires an environment to be configured. Check
out the [documentation](https://docs.aws.amazon.com/cdk/latest/guide/environments.html) for an overview and examples
```

Reply:
yeah i think this is a good idea! i think something simple like 

```ts
new MyDevStack(app, 'dev', { 
  env: { 
    account: process.env.CDK_DEFAULT_ACCOUNT, 
    region: process.env.CDK_DEFAULT_REGION 
}});
```
and some description of using profiles should be sufficient, what do you think?